lib/AuditLog/AuditLogCard/AuditLogCard.js

```diff
@@ -0,0 +1,120 @@
+import {
+  useMemo,
+  useState,
+} from 'react';
+import { Link } from 'react-router-dom';
+import {
+  FormattedMessage,
+  useIntl,
+} from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { Card } from '@folio/stripes/components';
+
+import { AuditLogChangedFieldsList } from '../AuditLogChangedFieldsList';
+import { AuditLogModal } from '../AuditLogModal';
+
+import { getChangedFieldsList } from '../getChangedFieldsList';
+import { formatDateTime } from '../../utils';
+
+import css from './AuditLogCard.css';
+
+const AuditLogCard = ({
+  date,
+  user,
+  diff,
+  fieldLabelsMap,
+  fieldFormatter,
+}) => {
+  const intl = useIntl();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  const anonymousUserLabel = intl.formatMessage({ id: 'stripes-acq-components.audit-log.anonymousUser' });
+
+  const headerStart = useMemo(() => {
+    return (
+      <b>
+        {formatDateTime(date, intl)}
+      </b>
+    );
+  }, [date, intl]);
+
+  const userName = useMemo(
+    () => {
+      return user ? `${user.personal.lastName}, ${user.personal.firstName}` : null;
+    }, [user],
+  );
+
+  const sourceUser = (
+    <FormattedMessage
+      id="stripes-components.metaSection.source"
+      tagName="p"
+      values={{
+        source: user
+          ? <Link to={`/users/preview/${user.id}`}>{userName}</Link>
+          : anonymousUserLabel,
+      }}
+    />
+  );
```

Comment:
@mariia-aloshyna I am glad you are thinking about this! I think components that know about specific applications do not belong in stripes-components, but that seems like a very small reason to kick this whole collection of components out. Could the user-link could be a prop instead of being hard-coded?

Reply:
Sure! I'll push PR with more unified version to stripes-components then, thank you!